iPokeGo/es.lproj/Localizable.strings

```diff
@@ -41,4 +41,4 @@
 "[Pokemon] was added to the map!" = "¡%@ ha sido añadido al mapa!";
 
 /* The hint that a favorite Pokémon appeared on the map. */
-"[Pokemon] your favorite pokemon was added to the map!" = "¡%@, tú Pokémon favorito ha sido añadido al mapa!";
+"[Pokemon] your favorite pokemon was added to the map!" = "¡%@, tu Pokémon favorito ha sido añadido al mapa!"";
```

Comment:
Cuidado que aquí hay un error con las últimas comillas, están puestas dos veces

"[Pokemon] your favorite pokemon was added to the map!" = "¡%@, tu Pokémon favorito ha sido añadido al mapa!";